x509/cert.go

```diff
@@ -12,21 +12,31 @@ func ReadCertificateFile(path string) ([]*x509.Certificate, error) {
 	if err != nil {
 		return nil, err
 	}
-	return ParseCertificatePEM(data)
+	return parseCertificates(data)
 }
 
-// ParseCertificatePEM parses a certificate PEM,
-// Does not perform any additional validations if the certs are a valid cert chain.
-func ParseCertificatePEM(data []byte) ([]*x509.Certificate, error) {
+// parseCertificates parses certificates from either PEM or DER data
+// returns an empty list if no certificates are found
+func parseCertificates(data []byte) ([]*x509.Certificate, error) {
```

Comment:
Can we add test cases for ReadCertificateFile if not present.

Reply:
We should add tests here. Some of the existing code in notation-go and core-go is missing test cases. Will address this separately. This code is being tested through trust-store parsing though.

Reply:
Sounds good, can you track this with an issue.